client/web/src/enterprise/batches/detail/changesets/BatchChangeChangesets.tsx

```diff
@@ -60,6 +62,8 @@ interface Props
         ExtensionsControllerProps,
         SettingsCascadeProps {
     batchChangeID: Scalars['ID']
+    batchChangeState: BatchChangeState
+    executionEnabled: boolean
```

Comment:
I know we're guilty of this in a number of places still, but we're trying to more consistently name our boolean variables/props with an `is` prefix, like `isExecutionEnabled`, so it's clearer that their value is a boolean even if you don't know the TS type.

 ```suggestion
    isExecutionEnabled: boolean
```

If you happen to spot other instances where we break this rule, feel free to correct them, too. 🙂

Reply:
No problem. I wasn't sure of the standard so thanks for clarifying.
I'll update this PR accordingly. Thanks.